blog/let's-roll.mdx

```diff
@@ -0,0 +1,59 @@
+---
+slug: Let's Roll- Rollkit's community call
+title: "Let's Roll- Rollkit's community call"
+authors: [AlCJoseph]
+tags: [Rollkit][Community]
+description: Rollkit is kicking off its community call, Let's Roll, to bring together community members to share knowledge, collaborate on new ideas, and inspire each other to push the limits of what's possible with Rollkit.
+---
+
+
+![Rollkit blog cover](../static/img/lets-roll/lets-roll.jpg)
+
+Last Sunday, we released the first [research implementation](https://rollkit.dev/blog/sovereign-rollups-on-bitcoin/) that allows Rollkit rollups to use Bitcoin for data availability. 
+
+[This announcement](https://twitter.com/RollkitDev/status/1632438374513676288) has sparked a lot of curiosity and excitement in the community. Many developers are starting to imagine what the future of sovereign rollups could look like on Bitcoin and other data availability layers. They’re looking for ways to innovate using this integration, to replicate it on other layer 1 blockchains, to identify challenges and to suggest improvements.
+ 
+We believe that there’s a lot of innovation to be done and a lot of challenges to be solved, and that together as a community, we can collaborate on building the future of rollup frameworks.
+
+That’s why we’re happy to announce that we’re launching a Rollkit’s community call: Let’s Roll.
+
+## TL;DR:
+
+-Rollkit is launching its community call, Let’s Roll. You can subscribe to the community calendar [here](https://calendar.google.com/calendar/u/0/embed?src=c_11a8d6a470ca9c07b131ab1ea213ac124a3061a9d6986bb208d88afbcc4bb3d2@group.calendar.google.com&ctz=America/Toronto)
+
+-Builders using Rollkit can now present their projects at Let’s Roll. Reach out to us on [Telegram](https://t.me/rollkit) 
```

Comment:
should the telegram link be join link instead of browse the telegram group?

Reply:
it could be but don't both links gets you to the same page on a laptop? only on phones it's different right?
here's the join link https://t.me/+THiNuoYyUEFjZjcx